src/app/(proper_react)/(redesign)/(public)/how-it-works/components/SectionOne.tsx

```diff
@@ -0,0 +1,155 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import { Props } from "../HowItWorksView";
+import {
+  RemoveStep1,
+  RemoveStep2,
+  RemoveStep3,
+  NameIcon,
+  LocationIcon,
+  DOBIcon,
+} from "../images";
+import styles from "../HowItWorksView.module.scss";
+import { TelemetryButton } from "../../../../../components/client/TelemetryButton";
+
+export const SectionOne = (props: Props) => {
+  const { l10n } = props;
+
+  return (
+    <div
+      id="data-brokers-section"
+      className={`${styles.sectionWrapper} ${styles.grayBG}`}
+    >
+      <div className={styles.intro}>
+        <span className={styles.introContent}>
+          <h2>
+            {l10n.getFragment("section-1-intro-title", {
+              elems: {
+                data_brokers: <span className={styles.brokersEmphasis} />,
+              },
+            })}
+          </h2>
+          <p>{l10n.getString("section-1-intro-text-1")}</p>
+          <hr className={styles.horizontalRule} />
+        </span>
+        <div className={styles.introCTA}>
+          <h3>{l10n.getString("section-1-intro-text-2")}</h3>
+          <div className={styles.sectionCTAButton}>
+            <TelemetryButton
+              variant="primary"
+              href="https://accounts.firefox.com/subscriptions/products/prod_OiV9RSaatywSRy?plan=price_1Nv4ODJNcmPzuWtRoYpoFHXd"
+              event={{
+                module: "ctaButton",
+                name: "click",
+                data: {
+                  button_id: "intent_to_purchase_first",
+                },
+              }}
+            >
+              {l10n.getString("section-1-intro-cta-button")}
+            </TelemetryButton>
+          </div>
+        </div>
+      </div>
+      {/* STEP 1 */}
+      <div className={styles.step}>
+        <div className={styles.stepTextContainer}>
+          <h3 className={`${styles.brokersEmphasis} ${styles.stepTitle}`}>
+            {l10n.getString("section-1-step-1-title")}
+          </h3>
+          <h3 className={styles.stepSubtitle}>
+            {l10n.getString("section-1-step-1-subtitle")}
+          </h3>
```

Comment:
Should this be an `<h4>` instead of `<h3>+<h3>`?

Reply:
h3's were in the Figma specs.  Is there a best practice I'm missing?